.github/workflows/build-push.yml

```diff
@@ -0,0 +1,87 @@
+name: build-push
+on:
+  push:
+    branches:
+      - main
+      - 'release-*'
```

Comment:
we can skip `release-` if we hard-code `devel` tag?

Reply:
true, I will remove. In this version we can push devel tag only. If we want to push release-xx versions, we need to provide mapping from branch name to tag, but lets handle that part in some next step.

Reply:
by same logic, we can also skip trigger on pull_request in this variant.

Reply:
but I still want to keep the "if" checks for these in code, as we will need those in final version (embedding this into CI flow). I hope the "if" part still works even when trigger source is removed...

Reply:
New version pushed, removed pull_request trigger, that likely will mean it is not tested here at all. But push anyway cant be tested in PR so thats ok.